rest/resource-server/src/main/java/org/eclipse/sw360/rest/resourceserver/report/SW360ReportController.java

```diff
@@ -93,17 +94,19 @@ public void getProjectReport(
         try {
             if (validateMimeType(mimeType)) {
                 switch (module) {
-                    case PROJECTS:
-                        getProjectReports(withLinkedReleases, mailRequest, response, request, sw360User, module, projectId);
-                        break;
-                    case COMPONENTS:
-                        getComponentsReports(withLinkedReleases, mailRequest, response, request, sw360User, module);
-                        break;
-                    case LICENSES:
-                        getLicensesReports(response, sw360User, module);
-                        break;
-                    default:
-                        break;
+                case PROJECTS:
+                    getProjectReports(withLinkedReleases, SW360Constants.MAIL_REQUEST_FOR_PROJECT_REPORT, response,
+                            request, sw360User, module, projectId);
+                    break;
+                case COMPONENTS:
+                    getComponentsReports(withLinkedReleases, SW360Constants.MAIL_REQUEST_FOR_COMPONENT_REPORT, response,
+                            request, sw360User, module);
+                    break;
+                case LICENSES:
```

Comment:
Do we need this case right now? as this PR is valid for projects and components only.

Reply:
It has no changes with previous commit. It is fine as it is.